chase/frontend/components/dashboard/documents/documenttemplate.tsx

```diff
@@ -0,0 +1,13 @@
+import React from "react";
+
+export default function DocumentTemplate({
```

Comment:
I'd probably rename to something like `DocumentWrapper` or `DocumentLayout` to make sure that this is purely visual and does not add any functionality but thats up to you

Reply:
In prime react, all custom visual wrappers that you can specify to make repetitive content look the same is called "Template".

I have absolutely no objection to change it, just wanted to highlight my reasoning for the name